lib/foreman_tasks/dynflow/daemon.rb

```diff
@@ -67,5 +87,50 @@ def run_background(command = 'start', options = {})
     def world
       ForemanTasks.dynflow.world
     end
+
+    private
+
+    def daemons_options(command, options)
+      {
+        :multiple => true,
+        :dir => options[:pid_dir],
+        :log_dir => options[:log_dir],
+        :dir_mode => :normal,
+        :monitor => true,
+        :log_output => true,
+        :monitor_interval => options[:memory_polling_interval] / 2,
+        :ARGV => [command]
+      }
+    end
+
+    def default_options
+      { foreman_root: Dir.pwd,
+        process_name: 'dynflow_executor',
+        pid_dir: "#{Rails.root}/tmp/pids",
+        log_dir: File.join(Rails.root, 'log'),
+        wait_attempts: 300,
+        wait_sleep: 1,
+        executors_count: (ENV['EXECUTORS_COUNT'] || 1).to_i,
+        memory_limit: ENV['EXECUTOR_MEMORY_LIMIT'].to_i,
+        memory_init_delay: (ENV['EXECUTOR_INIT_DELAY'] || 180).to_i,
```

Comment:
The 180s seems too short for me, I would suggest rather few hours (for some reason, 2 hours sound good to me:)